internal/exec/util/blkid.c

```diff
@@ -201,7 +201,9 @@ static result_t extract_part_info(blkid_partition part, struct partition_info *i
 	// label
 	ctmp = blkid_partition_get_name(part);
 	err = checked_copy(info->label, ctmp, PART_INFO_BUF_SIZE);
-	if (err)
+	if (err == RESULT_LOOKUP_FAILED)
```

Comment:
lets move this up so:
```
if (!ctmp)
    ctmp = "";
```
and add a comment that libblkid turns null if the label is an empty string.

Reply:
Done!